entries/s/swile.co.json

```diff
@@ -0,0 +1,17 @@
+{
+  "Swile": {
+    "domain": "swile.co",
+    "url": "https://www.swile.co/",
```

Comment:
```suggestion
```

The `url` attribute is only needed if going to the domain (e.g. `https://swile.co`) fails